package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@buildium/angular-ui",
-  "version": "1.0.1",
+  "version": "1.0.2-0",
```

Comment:
`npm version prerelease` bumped it to 1.0.2-0 ... @digitil which seems correct? I would've expected it to add a `-2`